### PR TITLE
empty objects in options have been replaced

### PR DESCRIPTION
**Pull request by abuDarda97**
This pull request solves [issue 7588](https://github.com/AnalyticalGraphicsInc/cesium/issues/7588). The object literals in `options = defaultValue(options, {});` have been replaced with `defaultValue.EMPTY_OBJECT`.

The CONTRIBUTORS.md file has been updated with my information.

### CODE REVIEW

**Comment by cesium-concierge** [comment]
Thanks for the pull request @abuDarda97!
* :x: Missing CLA.
    * Please send in a [Contributor License Agreement](https://github.com/AnalyticalGraphicsInc/cesium/blob/master/CONTRIBUTING.md#contributor-license-agreement-cla) (CLA) and comment back here to let us know to check this!
* :grey_question: [CHANGES.md](https://github.com/AnalyticalGraphicsInc/cesium/blob/options-updated/CHANGES.md) was not updated.
    * If this change updates the public API in any way, please add a bullet point to `CHANGES.md`.
* :grey_question: Unit tests were not updated.
    * Make sure you've [updated tests](https://github.com/AnalyticalGraphicsInc/cesium/tree/master/Documentation/Contributors/TestingGuide) to reflect your changes, added tests for any new code, and ran the code coverage tool.

Reviewers, don't forget to make sure that:

- [ ] Cesium Viewer works.
- [ ] Works in 2D/CV.
- [ ] Works (or fails gracefully) in IE11.

**Comment by abuDarda97** [comment]
Individual CLA text file has been sent.

**Comment by pjcozzi** [comment]
Thanks again for contributing, @abuDarda97, we received your CLA.

**Comment by abuDarda97** [comment]
@pjcozzi your welcome, I am glad that I can help.

File: Source/Renderer/Context.js [hunk 1]
```diff
@@ -191,7 +191,7 @@ define([
         this._canvas = canvas;
 
         options = clone(options, true);
-        options = defaultValue(options, {});
+        options = defaultValue(options, defaultValue.EMPTY_OBJECT);
```
Comment (shunter) [inline Source/Renderer/Context.js]: This change is incorrect because the line below modifies `options`.

Comment (abuDarda97) [reply Source/Renderer/Context.js]: So, I should change it back to `options = defaultValue(options, {});` ?

**Comment by cesium-concierge** [comment]
Thanks again for your contribution @abuDarda97! 

No one has commented on this pull request in 30 days. Maintainers, can you review, merge or close to keep things tidy?

 _I'm going to re-bump this in 30 days. If you'd like me to stop, just comment with `@cesium-concierge stop`. If you want me to start again, just delete the comment._

File: Source/Renderer/Context.js (hunk 1 above)
Comment (emackey) [reply Source/Renderer/Context.js]: I believe so, yes.  Please also put a comment pointing out that `options` is being modified, which explains why we can't use `EMPTY_OBJECT` here.

Comment (abuDarda97) [reply Source/Renderer/Context.js]: Hi @shunter @emackey I have made the changes, is there anything else I need to change?

**Comment by abuDarda97** [comment]
@OmarShehata

**Comment by cesium-concierge** [comment]
Thanks again for your contribution @abuDarda97! 

No one has commented on this pull request in 30 days. Maintainers, can you review, merge or close to keep things tidy?

 _I'm going to re-bump this in 30 days. If you'd like me to stop, just comment with `@cesium-concierge stop`. If you want me to start again, just delete the comment._

**Comment by OmarShehata** [comment]
Thanks for the review @shunter and @emackey !  This looks good to me, and can confirm all tests are running locally.

Thanks again @abuDarda97 !

**Pull request merged**